src/tests/App.test.js

```diff
@@ -1,16 +1,11 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import { create } from 'react-test-renderer';
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
 import App from "../App";
 
-it("renders without crashing", () => {
-  const div = document.createElement("div");
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
-});
-
 it("create a snapshot", () => {
```

Comment:
Line 12 should be `it(creates a snapshot)`

Reply:
I don't think it should be 'creates a snapshot'. The test is to 'create a snapshot'. It is more or less a test to check the interface.